pages/common/wp.md

```diff
@@ -0,0 +1,24 @@
+# wp
+
+> The official command-line interface to manage WordPress instances.
+> More information: <https://wp-cli.org/>.
+
+- Describe the operating system, shell, PHP, and WP-CLI (`wp`) installation:
+
+`wp --info`
+
+- Update WP-CLI:
+
+`wp cli update`
+
+- Install and activate a WordPress plugin:
+
+`wp plugin install {{plugin}} --activate`
+
+- Replace all instances of the old string in the database with the new string:
+
+`wp search-replace {{old}} {{new}}`
```

Comment:
```suggestion
- Replace all instances of a string in the database:

`wp search-replace {{old_string}} {{new_string}}`
```
I think "replace" already implies that it is replaced with something new. Also, I think more descriptive parameters would be helpful.

Reply:
I agree with this one, I did the original mostly because in the WordPress documentation that's how they described it. I figured maybe it was preferred for broader audiences. 🤔 

> Searches through all rows in a selection of tables and replaces appearances of the first string with the second string.
> \- https://developer.wordpress.org/cli/commands/search-replace/

I agree with the improvement, just wanted to note here why I did the original wording.